data/tooltips.txt

```diff
@@ -157,6 +157,9 @@ tip "disruption resistance heat:"
 
 tip "drag:"
 	`Limits the maximum speed of a ship.`
+	
+tip "drag reduction:"
+	`Reduces drag. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 50 percent reduction in drag, while a total value of 2 only results in a 66 percent reduction.`
```

Comment:
Note that protection attributes (which this is akin to) have their in-game display values multiplied by 100. This is so that a value of 1 as displayed to the user doesn't give such a massive buff of 50%. The same should be done here, and the tooltip would then be changed to look more like those of the protection attributes.

Reply:
Well it's not an attribute that protects you, so I don't know what you'd rename it to.